console error on navigation away from timeline object

<!--- Focus on user impact in the title. Use the Summary Field to -->
<!--- describe the problem technically. -->

#### Summary
Console error received when navigating away from a timeline object.
<!--- A description of the issue encountered. When possible, a description -->
<!--- of the impact of the issue. What use case does it impede?-->

#### Expected vs Current Behavior
No console errors on navigation away from timeline object.
<!--- Tell us what should have happened -->

#### Impact Check List
<!--- Please select from the following options -->

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug?

#### Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. navigate to a timeline object that exists, or create a new one
2. navigate to another object in tree
3.
4.

#### Environment
* Open MCT Version: <!--- date of build, version, or SHA -->
* Deployment Type: <!--- npm dev? VIPER Dev? openmct-yamcs? -->
* OS:
* Browser:

#### Additional Information
<!--- Include any screenshots, gifs, or logs which will expedite triage -->
```
vue.min.js:6 Error: Attempted to destroy non-mutable domain object
    at g.destroyMutable (ObjectAPI.js:265)
    at a.beforeDestroy (TimelineObjectView.vue:92)
    at me (vue.min.js:6)
    at a.e.$destroy (vue.min.js:6)
    at destroy (vue.min.js:6)
    at m (vue.min.js:6)
    at m (vue.min.js:6)
    at m (vue.min.js:6)
    at m (vue.min.js:6)
    at m (vue.min.js:6)
```

--- a/src/plugins/persistence/couch/CouchObjectProvider.js
+++ b/src/plugins/persistence/couch/CouchObjectProvider.js
@@ -555,15 +555,18 @@ class CouchObjectProvider {
         let intermediateResponse = this.getIntermediateResponse();
         const key = model.identifier.key;
         this.enqueueObject(key, model, intermediateResponse);
-        this.objectQueue[key].pending = true;
-        const queued = this.objectQueue[key].dequeue();
-        let document = new CouchDocument(key, queued.model);
-        this.request(key, "PUT", document).then((response) => {
-            this.checkResponse(response, queued.intermediateResponse, key);
-        }).catch(error => {
-            queued.intermediateResponse.reject(error);
-            this.objectQueue[key].pending = false;
-        });
+        if (!this.objectQueue[key].pending) {
+            this.objectQueue[key].pending = true;
+            const queued = this.objectQueue[key].dequeue();
+            let document = new CouchDocument(key, queued.model);
+            this.request(key, "PUT", document).then((response) => {
+                console.log('create check response', key);
+                this.checkResponse(response, queued.intermediateResponse, key);
+            }).catch(error => {
+                queued.intermediateResponse.reject(error);
+                this.objectQueue[key].pending = false;
+            });
+        }
 
         return intermediateResponse.promise;
     }

--- a/src/plugins/timeline/TimelineObjectView.vue
+++ b/src/plugins/timeline/TimelineObjectView.vue
@@ -88,7 +88,7 @@ export default {
             this.mutablePromise.then(() => {
                 this.openmct.objects.destroyMutable(this.domainObject);
             });
-        } else {
+        } else if (this.domainObject.isMutable) {
             this.openmct.objects.destroyMutable(this.domainObject);
         }
     },

